pydra/engine/core.py

```diff
@@ -504,13 +504,8 @@ def add(self, task):
             if isinstance(val, LazyField):
                 # adding an edge to the graph if task id expecting output from a different task
                 if val.name != self.name:
+                    self.graph.add_edges((getattr(self, val.name), task))
```

Comment:
this doesn't appear to be storing the specific edge info. the output and input names. where is that being stored?

Reply:
but for sorting graph etc. we don't need this info. right?

Reply:
since the graph is the workflow, it would be useful to store that information here rather than somewhere else. the link to the nodes feeding information to a node is right here. if not this information is also being stored in the node. seems redundant.

Reply:
ok, can store it with the graph and see which wf attributes could be removed

Reply:
we agreed that it's not needed. everything is stored in LazyField